docs/en_US/ResearchBlog/NASComparison.md

```diff
@@ -0,0 +1,81 @@
+# NAS Algorithms Comparison
+Train and Compare NAS models including Autokeras, DARTS, ENAS and NAO.
+
```

Comment:
do you want to add a conclusion here?

Reply:
I don't want to get a definitive conclusion here and I just compare them in different datasets.